sentry/projects.go

```diff
@@ -52,7 +52,6 @@ type Project struct {
 
 	Organization Organization `json:"organization"`
 	// TODO: plugins
-	// TODO: platforms
```

Comment:
Is the `platforms` https://github.com/jianyuan/go-sentry/pull/5/files#diff-612f6a10f38bbf357da2bdad99b61b84L26?